keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -86,20 +86,29 @@ As of 29-11-2018 much of the work for enabling Windows nodes has already been co
-    - Pod (single or multiple containers per Pod with process isolation)
+    - Pod (single or multiple containers per Pod with process isolation). There are no notable differences in Pod status fields between Linux and Windows containers
     - Services types NodePort, ClusterIP, LoadBalancer, and ExternalName
     - Workload controllers ReplicaSet, ReplicationController, Deployments, StatefulSets, DaemonSet, Job, CronJob
     - ConfigMap, Secrets: as environment variables or volumes
     - Resource limits
     - Pod & container metrics
     - Horizontal Pod Autoscaling
+    - Volumes can be shared between containers in a Pod
+    - EmptyDir
 - Windows Server 2019 is the only Windows operating system we will support at GA timeframe. Note above that the host operating system version and the container base image need to match. This is a Windows limitation we cannot overcome.
 - Customers can deploy a heterogeneous cluster, with Windows and Linux compute nodes side-by-side and schedule Docker containers on both operating systems. Of course, Windows Server containers have to be scheduled on Windows and Linux containers on Linux
 - Out-of-tree Pod networking with [Azure-CNI](https://github.com/Azure/azure-container-networking/blob/master/docs/cni.md), [OVN-Kubernetes](https://github.com/openvswitch/ovn-kubernetes), [two CNI meta-plugins](https://github.com/containernetworking/plugins), [Flannel (VXLAN and Host-Gateway)](https://github.com/coreos/flannel) 
 - Dockershim CRI
 - Many<sup id="a1">[1]</sup> of the e2e conformance tests when run with [alternate Windows-based images](https://hub.docker.com/r/e2eteam/) which are being moved to [kubernetes-sigs/windows-testing](https://www.github.com/kubernetes-sigs/windows-testing)
 - Persistent storage: FlexVolume with [SMB + iSCSI](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows), and in-tree AzureFile and AzureDisk providers
- 
+
+### What we need to test and verify if it works or not for GA (items that are unsupported will be documented)
+- Headless services (https://github.com/kubernetes/kubernetes/issues/73416)
+- OOM reporting (https://github.com/kubernetes/kubernetes/issues/73417)
+- QoS (guaranteed, burstable, best effort) (https://github.com/kubernetes/kubernetes/issues/73418)
+- Is terminationGracePeriodSeconds supported (https://github.com/moby/moby/issues/25982)
+- Pod DNS configuration like hostname, subdomain, hostAliases, dnsConfig, dnsPolicy (https://github.com/kubernetes/kubernetes/issues/73414)
+
 ### Windows Node Roadmap (post-GA work)
 - Group Managed Service Accounts, a way to assign an Active Directory identity to a Windows container, is forthcoming with KEP `Windows Group Managed Service Accounts for Container Identity`
 - `kubectl port-forward` hasn't been implemented due to lack of an `nsenter` equivalent to run a process inside a network namespace.
```

Comment:
Do exec-based mechanisms work, such as liveness and readiness probes, postStart and preStop lifecycle hooks, and kubectl exec?

Reply:
`kubectl exec` works, readiness and liveness work

Reply:
Is it that these features of Kubernetes are just not tested, or that the existing tests are all Linux-specific?